crucible-install.sh

```diff
@@ -23,14 +23,19 @@ EC_FAIL_INSTALL=5
 EC_AUTH_FILE_NOT_FOUND=6
 EC_FAIL_DEPENDENCY=7
 EC_FAIL_REGISTRY_UNSET=8
-#9
+EC_TLS_VERIFY_ERROR=9
 EC_INVALID_OPTION=10
 EC_UNEXPECTED_ARG=11
 EC_FAIL_REGISTRY_SET=12
 EC_FAIL_AUTH_SET=13
 EC_FAIL_CHECKOUT=14
 EC_PUSHD_FAIL=15
 
+# remove a previous installation log
+if [ -e ${GIT_INSTALL_LOG} ]; then
+    rm ${GIT_INSTALL_LOG}
+fi
+
```

Comment:
why? what does it have to do with tls ?

Reply:
Nothing.  Except I was initially getting a totally misleading error log because it was discovering an old one :(